docker-compose.yml

```diff
@@ -9,16 +9,17 @@ services:
       DEBUG_MODE: "false"
       ELASTIC_APM_SERVER_URL: "http://localhost:8200"
       ELASTICSEARCH_URL: "http://localhost:9200"
-      KARTOFFEL_URL: "http://kartoffel-master.eastus.cloudapp.azure.com:3001"
```

Comment:
I prefer to use Kartoffel with Spike here to refrain from errors that could occur later.

Reply:
done.